README.md

```diff
@@ -33,6 +33,14 @@ The Nuke Adaptor implements the [OpenJD][openjd-adaptor-runtime] interface that
 
 Jobs created by the submitter use this adaptor by default.
 
+### Compatibility
+
+This library requires:
+
+1. Nuke 15,
+1. Python 3.9 or higher; and
+1. Linux operating system.
```

Comment:
The adaptor should work on macos, linux, and windows? Although maybe Evan's PR fixes Windows, i'd have to double check. This _should_ be the same compatibility as the submitter currently.

Reply:
I see. I will revert the changes I have in README. Thanks!